data/sql/updates/pending_db_world/Asp_Npcs_Flags_change.sql

```diff
@@ -0,0 +1,30 @@
+
+UPDATE `creature_template` SET `unit_flags` = `unit_flags` & ~(256|512) WHERE `entry` = 29072;
+UPDATE `creature_template` SET `unit_flags` = `unit_flags` |2|131072 WHERE `entry` = 29072;
```

Comment:
These changes do not match sniffs, observed value in a 9.1.5 41079 sniff (when walking up to the creature before the RP begins, so probably default flags) is 33536 (256+512+32768) as it currently is on AC.